src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,75 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React, { useCallback, useState } from 'react';
+import debounce from 'lodash.debounce';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  selectFilter: string;
+  searchFilter: string;
+  onSelectFilter: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
also one of the possible variants 

```suggestion
  onSelectFilter: (value: string) => void;
```